src/MIDebugEngine/Engine.Impl/Breakpoints.cs

```diff
@@ -86,11 +86,15 @@ private static MIBreakpointState StringToBreakpointState(string addr)
 
         internal static async Task<BindResult> Bind(string functionName, DebuggedProcess process, string condition, AD7PendingBreakpoint pbreak)
         {
+            process.VerifyNotDebuggingCoreDump();
```

Comment:
what happens if this throws from the user's perspective in vscode? In vs, we'd display an error breakpoint in the gutter that says that bps aren't supported when dump debugging.